Formula/pypy3@3.8.rb

```diff
@@ -0,0 +1,222 @@
+class Pypy3AT38 < Formula
```

Comment:
I'm not a fan of `pypy3@3.8`, especially since `@3.8` doesn't give the version of `pypy3` but the version of Python3 that this implements.

How about `pypy38` or `pypy3.8`?

Reply:
How do you feel about my comments here: <https://github.com/Homebrew/homebrew-core/pull/88123#discussion_r746238241>?

By analogy, what happens when e.g. Truffleruby ends up in Homebrew, and it supports both 3.1 and 3.0? What if Luajit starts supporting 5.1 and 5.4 (not that it would ever happen)? Whatever we decide for PyPy should also make sense for other things, IMO.

Reply:
I would say we definitely don't want to use `@` in those cases either, because the convention is that whatever comes after the `@` is the version of the software.

Reply:
Noted. In that case I will rename this to `pypy3.8`.

I believe they intend to support 3.7 and 3.8 concurrently for a while. They also aren't going to drop Python 2 support for a while, if ever (unlike CPython core).

Does that change anything in your assessment?

Maybe the full setup should be something like this? Or is it too complicated?

* `pypy2.7`
* `pypy3.7`
* `pypy3.8`
* `pypy3` as an alias for `pypy3.7` (changing to `pypy3.8` when it is no longer in beta)
* `pypy2` as an alias for `pypy2.7`
* `pypy` as an alias for `pypy2` for backward compatibility

Reply:
Any chance this will be released soon?

Reply:
It has the wrong name and the author never complied to the request to change it, AFAICT.

Reply:
Maybe this activity will bring the author back here.

Reply:
The PR has been stale for a while; feel free to open an alternative that incorporates feedback from here. You can try cherry-picking commits from here if you'd like to share credit (in the commit history, at least) with @gwerbin.

Reply:
Apologies, @ulfarsson I've been busy with other things. I will try to remember to do it this afternoon.

Reply:
Beautiful, thank you :).

Reply:
Thanks, @gwerbin. After thinking about it some more, I'm much more partial to `pypy38` over `pypy3.8`. The `38` is less likely to be misconstrued as the version of `pypy` than `3.8` in `pypy3.8` (especially for those not so familiar with the `@` versioning convention).

Reply:
I would suggest sticking with `pypy3.8` as that's the naming convention used by PyPy itself, see for example https://www.pypy.org/download.html and https://downloads.python.org/pypy/

Reply:
@axelarge, that's fine with me too then.

Reply:
How do I correctly specify the name `pypy3.8` in a formula? What are the correct filename and class name?